deploy/charts/rook-ceph-cluster/prometheus/localrules.yaml

```diff
@@ -525,6 +529,15 @@ groups:
           type: "ceph_default"
   - name: "pools"
     rules:
+      - alert: "CephPoolGrowthWarning"
```

Comment:
The pool growth warning was causing issues as seen in #10070, so we had removed it from Rook. Is the rule changed now? It may need more investigation if we are going to add it back.

Reply:
https://github.com/ceph/ceph/commit/97189b66afd4623ae09bc4ba12e1af6f69821793 has resolved the issue

As you can see in https://github.com/rook/rook/issues/10070#issuecomment-1099455108, the expression contained `on(pool_id) group_right()` which caused a `many-to-many matching not allowed` error. The above fix added an `instance` filtering into the expression that resolved the issue.